prometheus/label.go

```diff
@@ -17,12 +19,12 @@ func (l label) hash() uint64 {
 	return h
 }
 
-func (l1 label) equal(l2 label) bool {
```

Comment:
This was done to make the receiver names consistent.  It is not part of the logic concerning this PR.  I can reverse these chunks if there is any objection.